Database/Corrections/sodQuestFixes.lua

```diff
@@ -3168,6 +3168,30 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410107,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90216] = {
+            [questKeys.name] = "Molten Blast",
+            [questKeys.startedBy] = {{208275}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Use fire spells to get 5 stacks of debuff on the Frozen Makura then loot it and equip the Icon and follow its instructions to receive the rune."},
```

Comment:
I don't think you can do this rune on your own, just like the other runes where you need to attack some frozen NPC with fire spells.

For the "Burnout" rune we (yeah, me) added the following text:

> Gather some Mage and Warlock friends and attack the Frozen Makrura with fire spells.

I am open for suggestions, but should streamline the wording then I guess.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5611/commits/41d028fb6a8d634631939db7bb3efe53f6abc5ef

I think it's okay to have this as the line right now, but maybe we should adjust it based on user feedback?